package.json

```diff
@@ -38,7 +38,7 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "express": "4.17.1",
+    "express": "4.17.2",
```

Comment:
If express is only used for the typing, it seems not necessary for the package usage.